CHANGELOG.md

```diff
@@ -121,8 +121,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 - Initial release
 
-[Unreleased]: https://github.com/GoogleCloudPlatform/functions-framework-python/compare/v2.4.0-beta.1...HEAD
```

Comment:
I believe this should be: https://github.com/GoogleCloudPlatform/functions-framework-python/compare/v2.4.0-beta.2...HEAD